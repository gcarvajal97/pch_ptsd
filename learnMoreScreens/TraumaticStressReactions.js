import i18n from 'i18n-js';
import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView, Dimensions, AccessibilityInfo} from 'react-native';
import { Video } from 'expo-av';
import translate, { getVideo } from '../components/translateService';
import color from '../colors';

const videos = {
  en: require("../assets/videos/vidStressReactions.mp4"),
  es: require("../assets/videos/vidStressReactionsEs.mp4"),
};

class TraumaticStressReactions extends Component {
  
  static navigationOptions = {
    headerTitle: translate('traumaticStressReactions.header')
  }

  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.title}>
          {translate('traumaticStressReactions.title')}
        </Text>
        <Video
            source={getVideo(videos)}
            rate={1.0}
            volume={1.0}
            resizeMode={Video.RESIZE_MODE_STRETCH}
            resizeMode={Video.RESIZE_MODE_COVER}
            shouldPlay
            useNativeControls
            style={{ 
              height: 204, 
              width: '100%', 
              maxHeight: 254, 
              borderColor: color.accent, 
              borderWidth: 14, 
              borderTopWidth: 0, 
              borderBottomWidth: 0, 
              marginBottom:-1
            }}
            accessible
            accessibilityLabel={translate(
              'traumaticStressReactions.content.videoCard.accessibility'
              )}
            accessibilityHint={translate(
              'traumaticStressReactions.content.videoCard.accessibility'
              )}
        />
        <Text style={styles.subTitle}>
          <Text style={{ fontWeight: 'bold' }}>{translate('traumaticStressReactions.content.videoCard.title')}</Text>
          {translate('traumaticStressReactions.content.videoCard.subtitle')}
        </Text>
        <ScrollView>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.bulletListOne.title')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.bulletListOne.bulletOne')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.bulletListOne.bulletTwo')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.bulletListOne.bulletThree')}</Text>
          <Text style={styles.paragraph}>
          <Text style={{ fontWeight: 'bold' }}>{translate('traumaticStressReactions.content.reExperiencingParagraph.boldNote')}</Text>
          {translate('traumaticStressReactions.content.reExperiencingParagraph.note')}</Text>
          <Text style={styles.paragraphTitle}>{translate('traumaticStressReactions.content.reExperiencingParagraph.title')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.reExperiencingParagraph.paragraphOne')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.reExperiencingParagraph.paragraphTwo')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.reExperiencingParagraph.paragraphThree')}</Text>
          <Text style={styles.paragraphTitle}>{translate('traumaticStressReactions.content.avoidanceParagraph.title')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.avoidanceParagraph.paragraphOne')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.avoidanceParagraph.paragraphTwo')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.avoidanceParagraph.paragraphThree')}</Text>
          <Text style={styles.paragraphTitle}>{translate('traumaticStressReactions.content.hyperArousalParagraph.title')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.hyperArousalParagraph.paragraphOne')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListOne.bulletOne')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListOne.bulletTwo')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.hyperArousalParagraph.paragraphTwo')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListTwo.title')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListTwo.bulletOne')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListTwo.bulletTwo')}</Text>
          <Text style={styles.paragraph}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListThree.title')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListThree.bulletOne')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListThree.bulletTwo')}</Text>
          <Text style={styles.bullet}>{translate('traumaticStressReactions.content.hyperArousalParagraph.bulletListThree.bulletThree')}</Text>
        </ScrollView>
    </View>
    );
  }
}

export default TraumaticStressReactions;

const styles = StyleSheet.create({
    container: {
      flex: 1,
      alignItems: 'center',
      margin: 10,
      backgroundColor: colors.primary.white,
      shadowColor: colors.primary.black,
      shadowOffset: {
          width: 0,
          height: 2,
      },
      shadowOpacity: 0.23,
      shadowRadius: 2.62,

      elevation: 4,
  },
  title: {
      backgroundColor: colors.accent, 
      color: colors.primary.white, 
      fontWeight: 'bold', 
      fontSize: 22, 
      textAlign: 'center', 
      alignSelf: 'stretch'
  },
  subTitle: {
      backgroundColor: colors.accent, 
      color: colors.primary.white, 
      alignSelf: 'stretch', 
      paddingLeft: 15, 
      paddingRight: 15, 
      fontSize: 15, 
      borderBottomWidth: 2, 
      borderBottomColor: colors.accent,
  },
  paragraphTitle: {
      fontWeight: 'bold',
      fontSize: 20,
      alignSelf: 'stretch',
      textAlign: 'left',
      marginLeft: 5,
      marginTop: 4,
      marginBottom: 3,
      color: colors.accent
  },
  bullet: {
      marginVertical: 1, 
      marginHorizontal: 18, 
      textAlign: 'left', 
      alignSelf: 'stretch', 
      fontWeight: 'bold',
      fontSize: 14
  },
  paragraph: {
      alignSelf: "center", 
      paddingVertical: 1, 
      paddingHorizontal: 5,
      fontSize: 14,
      marginTop: 5,
  },
  link: {
      alignSelf:"center", 
      paddingVertical: 1, 
      paddingHorizontal: 5,
      fontSize: 16,
      color: colors.accent,
      textDecorationLine: 'underline',
      marginTop: 4,
      marginTop: 3,
  }
});