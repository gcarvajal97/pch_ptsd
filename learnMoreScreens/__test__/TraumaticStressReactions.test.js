import React from 'react';
import TraumaticStressReactions from '../TraumaticStressReactions';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<TraumaticStressReactions />).toJSON();
    expect(tree).toMatchSnapshot();
})