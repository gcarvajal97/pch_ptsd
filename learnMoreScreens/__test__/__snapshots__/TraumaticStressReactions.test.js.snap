// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TraumaticStressReactions renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "elevation": 4,
      "flex": 1,
      "margin": 10,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "color": "#FFFFFF",
        "fontSize": 22,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    Traumatic Stress Reactions
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "borderBottomWidth": 0,
        "borderColor": "#0077C8",
        "borderTopWidth": 0,
        "borderWidth": 14,
        "height": 204,
        "marginBottom": -1,
        "maxHeight": 254,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoVideoView
      accessibilityHint="Video on what are traumatic stress reactions."
      accessibilityLabel="Video on what are traumatic stress reactions."
      accessible={true}
      proxiedProperties={
        Object {
          "onError": [Function],
          "onFullscreenUpdate": [Function],
          "onLoad": [Function],
          "onLoadStart": [Function],
          "onReadyForDisplay": [Function],
          "onStatusUpdate": [Function],
          "resizeMode": "mock",
          "source": Object {
            "headers": undefined,
            "overridingExtension": null,
            "uri": "uri",
          },
          "status": Object {
            "rate": 1,
            "shouldPlay": true,
            "volume": 1,
          },
          "useNativeControls": true,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "borderBottomColor": "#0077C8",
        "borderBottomWidth": 2,
        "color": "#FFFFFF",
        "fontSize": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      Stress reactions to injury are normal
    </Text>
     However, three major reactions should be monitored.
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        There are three main types of traumatic stress reactions:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Re-experiencing: Reliving what happened
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Avoidance: Staying away from reminders
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Hyper-arousal: Feeling anxious or jumpy
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          Note: 
        </Text>
        These reactions are normal and are not considered post traumatic stress reactions or PTSD unless they persist for over a month and interfere with daily life.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Re-Experiencing
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Re-experiencing means that your child keeps thinking a lot about the trauma, even when they do not want to.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Some re-experiencing is normal and natural. Thinking a lot about what happened, especially at first, is part of how we help ourselves recover from a scary experience.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Too much re-experiencing can be very distressing. Your child might have nightmares or “flashbacks” that make them feel like they are going through the trauma all over again. They may feel really upset or even have physical symptoms when something – a sight, a sound, a smell – reminds them of what happened.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Avoidance
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Avoidance symptoms can start by trying not to think or talk about the trauma, or anything connected with it. Sometimes kids want to stay away from people, places or activities because these reminders upset them. And children sometimes develop new fears or worries.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Of course, it can be good common sense to be more cautious after an injury. Children may become more aware of safety -- remembering to wear a seat belt, not running into the street after a ball, or staying away from dogs they do not know.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        But extreme avoidance or fears can become a real problem. Avoidance can interfere with daily life and stop your child from getting back to enjoying things that they usually like to do.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Hyper-Arousal
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Hyper-arousal also starts with a natural and normal response to danger – the "fight or flight response." After an injury or accident, this "fight or flight response." response might not turn off, even when you are safe.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Your heart keeps pounding and you start sweating.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Your body is still on the lookout for danger.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        After a scary situation like being injured, your child might have the feeling that something bad could happen again at any time, or might jump at any loud noise.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        The physical feelings that go along with hyper-arousal can feel scary themselves:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Your child might suddenly feel their heart racing or head pounding.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - They might get worried about what these symptoms mean.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Feeling “on guard” for danger much of the time can lead to:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - trouble sleeping
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - trouble concentrating
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - being extra cranky or irritable
      </Text>
    </View>
  </RCTScrollView>
</View>
`;

exports[`TraumaticStressReactions renders correctly in Spanish 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "elevation": 4,
      "flex": 1,
      "margin": 10,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "color": "#FFFFFF",
        "fontSize": 22,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    Reacciones de estrés traumático
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "borderBottomWidth": 0,
        "borderColor": "#0077C8",
        "borderTopWidth": 0,
        "borderWidth": 14,
        "height": 204,
        "marginBottom": -1,
        "maxHeight": 254,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoVideoView
      accessibilityHint="Video sobre qué son las reacciones de estrés traumático."
      accessibilityLabel="Video sobre qué son las reacciones de estrés traumático."
      accessible={true}
      proxiedProperties={
        Object {
          "onError": [Function],
          "onFullscreenUpdate": [Function],
          "onLoad": [Function],
          "onLoadStart": [Function],
          "onReadyForDisplay": [Function],
          "onStatusUpdate": [Function],
          "resizeMode": "mock",
          "source": Object {
            "headers": undefined,
            "overridingExtension": null,
            "uri": "uri",
          },
          "status": Object {
            "rate": 1,
            "shouldPlay": true,
            "volume": 1,
          },
          "useNativeControls": true,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "borderBottomColor": "#0077C8",
        "borderBottomWidth": 2,
        "color": "#FFFFFF",
        "fontSize": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      Las reacciones de estrés a la lesión son normales.
    </Text>
     Sin embargo, se deben monitorear tres reacciones principales.
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Hay tres tipos principales de reacciones de estrés traumático:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Volver a experimentar: revivir lo que pasó
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Evitación: mantenerse alejado de los recordatorios
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Hiperexcitación: sentirse ansioso o nervioso
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          Nota: 
        </Text>
        Estas reacciones son normales y no se consideran reacciones de estrés postraumático o TEPT a menos que persistan durante más de un mes e interfieran con la vida diaria.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Volver a experimentar
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Volver a experimentar significa que su hijo sigue pensando mucho sobre el trauma, incluso cuando no quiere.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Volver a experimentar un poco es normal y natural. Pensar mucho sobre lo sucedido, sobre todo al principio, es parte de cómo nos ayudamos a nosotros mismos a recuperarnos de una experiencia aterradora.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Volver a experimentar demasiado puede ser muy angustiante. Su hijo podría tener pesadillas o "escena retrospectiva" que le hagan sentir que está pasando por el trauma de nuevo. Tal vez se sienta muy angustiado o incluso tenga síntomas físicos cuando algo (una visión, un sonido, un olor) le recuerde lo que pasó.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Evitación
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Los síntomas de evitación pueden empezar por tratar de no pensar o hablar sobre el trauma o cualquier cosa relacionada con él. A veces los niños quieren mantenerse alejados de personas, lugares o actividades debido a que estos recordatorios los perturban. Y a veces los niños desarrollan nuevos temores o preocupaciones.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Por supuesto, el ser más cauteloso después de una lesión puede indicar un buen sentido común. Puede que los niños se hagan más conscientes de la seguridad, recordando usar el cinturón de seguridad, no corriendo a la calle detrás de una pelota o manteniéndose alejados de los perros que no conocen.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Pero la evitación o los temores extremos pueden llegar a ser un verdadero problema. La evitación puede interferir con la vida diaria y evitar que su hijo vuelva a disfrutar de las cosas que normalmente le gusta hacer.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Hiperexcitación
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        La hiperexcitación también comienza con una respuesta natural y normal al peligro: la "reacción de lucha o huida". Después de una lesión o accidente, esta respuesta de "lucha o huida" puede que no se desactive, incluso cuando uno está seguro.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Su corazón sigue palpitando y tiene sudores.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Su cuerpo todavía está atento al peligro.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Después de una situación de miedo como la de estar herido, su hijo puede tener la sensación de que algo malo podría ocurrir de nuevo en cualquier momento o puede saltar ante cualquier ruido fuerte.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Las sensaciones físicas que acompañan a la hiperexcitación pueden dar miedo en sí mismas:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Su hijo tal vez se sienta de repente con el corazón acelerado o la cabeza retumbándole.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Él podría preocuparse por lo que significan estos síntomas.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Sentirse "en guardia" por el peligro la mayor parte del tiempo puede llevar a:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Problemas para dormir
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Problemas para concentrarse y
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Tener un mayor mal humor o estar más irritable que lo habitual.
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
